turbo.json

```diff
@@ -0,0 +1,18 @@
+{
+  "$schema": "https://turborepo.org/schema.json",
+  "baseBranch": "origin/master",
+  "pipeline": {
+    "test": {
+      "dependsOn": ["^build"],
+      "outputs": []
+    },
+    "build": {
+      "dependsOn": ["test"],
+      "outputs": ["out/**", "dist/**"]
+    },
+    "frontend-app#build": {
+      "dependsOn": ["backend#build", "test"],
```

Comment:
`frontend-app` doesn't explicitly depend on `backend`, so we have to make a connection here.  In a real world example, this wouldn't be tied to the build, but perhaps a `deploy` job.